### PR TITLE
[DO NOT MERGE] Add hybrid changes

### PR DESCRIPTION
**Pull request by nemijato**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [x] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet. - During March
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet. Estimate September.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [x] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by ericshape** [comment]
LGTM

**Review by ericshape — APPROVED** [review]
LGTM

**Comment by ericshape** [comment]
@akning-ms 
Please hold on and not merge until @nemijato and I decide the merging schedule. Thanks!

**Review by bcham — APPROVED** [review]

**Comment by chiragg4u** [comment]
Please go through the contribution checklist added as part of the review and mark them completed. It will help you fix most of the issues and improve the quality of swagger and help close the PR faster. This is a requirement before review can be continued.

**Comment by akning-ms** [comment]
> @akning-ms
> Please hold on and not merge until @nemijato and I decide the merging schedule. Thanks!

Got it, next time, you can a label "DoNotMerge" and remove it when it is ready for merge

**Comment by ericshape** [comment]
> > @akning-ms
> > Please hold on and not merge until @nemijato and I decide the merging schedule. Thanks!
> 
> Got it, next time, you can a label "DoNotMerge" and remove it when it is ready for merge

oh. I just know this tag 👍 
Thanks!

**Comment by ericshape** [comment]
> Please go through the contribution checklist added as part of the review and mark them completed. It will help you fix most of the issues and improve the quality of swagger and help close the PR faster. This is a requirement before review can be continued.

@chiragg4u 
The track2 validation issue is not introduced by this PR. 
I will fix the track2 validation issue when I merge https://github.com/Azure/azure-rest-api-specs/tree/dev-sql-Microsoft.Sql-2020-11-01-preview into the master branch.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/HybridLinks.json
Comment (j5lim) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/HybridLinks.json:47]: >default": { [](start = 11, length = 11)

Please add the schema for error. The same comment applies to other operations as well.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/HybridLinks.json
Comment (j5lim) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/HybridLinks.json:154]: Using 201 + provisioning state + Azure-asyncOperation header is recommended for async PUT.
[https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc0025-201-is-the-recommended-async-pattern](https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc0025-201-is-the-recommended-async-pattern)

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/HybridLinks.json
Comment (j5lim) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/HybridLinks.json:298]: RPC requires systemData for new api-version.
[https://github.com/Azure/azure-resource-manager-rpc/blob/f39512268c61b0a970766515d535a756495bb56d/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources](https://github.com/Azure/azure-resource-manager-rpc/blob/f39512268c61b0a970766515d535a756495bb56d/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources)
You can ref the schema in the common type json.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/examples/HybridLinkUpdate.json
Comment (j5lim) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/examples/HybridLinkUpdate.json:24]: The schema for this is not defined. 202 should return an empty body with headers.

**Review by j5lim — CHANGES_REQUESTED** [review]
left comments

**Pull request closed without merging**
